tools/test/travis-ci/scancode-evaluate.py

```diff
@@ -112,7 +125,7 @@ def license_check(scancode_output_path):
             # check the next file in the scancode output
             continue
 
-        if not has_permissive_text_in_scancode_output(scancode_output_data_file['licenses']):
+        if not has_permissive_text_in_scancode_output(scancode_output_data_file):
```

Comment:
This was probably ok as it was.
You probably want to do something like:
```suggestion
        if (
            not has_permissive_text_in_scancode_output(scancode_output_data_file['licenses'])
            or not has_binary_license(scanned_file_content)
        ):
```